providers/song-controls.js

```diff
@@ -12,8 +12,7 @@ module.exports = (win) => {
 		// Playback
 		previous: () => pressKey(win, "k"),
 		next: () => pressKey(win, "j"),
-		playPause: () => win.webContents.send("playPause"),
-		like: () => pressKey(win, "_"),
+		playPause: (toPlay = undefined) => win.webContents.send("playPause", toPlay),
```

Comment:
I'm getting a `Error: Failed to serialize arguments` (`song-controls.js:15:54`) when using the play/pause media key, is it expected?

Reply:
It happened because shortcut plugin call the method with `win.webContents` arguement.
I just fixed it by making `play, pause` directly part of song controls
sorry about that